build_tools/mako/config/benchmark_template.config

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This template is used to define/update mako benchmarks. The benchmark_key,
+# benchmark_name, and metric_info_list should be filled. See:
```

Comment:
`benchmark_name` and `metric_info_list` aren't listed here. How about putting in examples? Also people should update the description, so probably note that.

Reply:
Done, and add more comments!